population/population_types.go

```diff
@@ -42,7 +44,12 @@ func (c *Client) GetPopulationTypes(ctx context.Context, input GetPopulationType
 		"method": http.MethodGet,
 	}
 
-	urlPath := "population-types"
+	limit := 1000
```

Comment:
Do we want it to be an enormously high no. for future proofing

Reply:
I've defaulted to passing no parameters but I suspect we are now having to pass in the number 1000 in every place we use the method

Reply:
or perhaps 10,000 as you suggest - though I think if we get over 100 our servers will burst into fire